### ISSUE TITLE
Can't clone repository on windows 7

### ISSUE THREAD

**Issue by stiv-yakovenko** [git-for-windows/git#1136]
$ git --version --build-options

git version 2.12.0.windows.1
built from commit: d4baef1cdfef9de62d43f9682a98ba57adc842e4
sizeof-long: 4
machine: x86_64

$ cmd.exe /c ver
Microsoft Windows [Version 6.1.7601]

```
> type "C:\Program Files\Git\etc\install-options.txt"

Path Option: Cmd
SSH Option: OpenSSH
CRLF Option: CRLFAlways
Bash Terminal Option: MinTTY
Performance Tweaks FSCache: Enabled
Use Credential Manager: Enabled
Enable Symlinks: Disabled

C:/tmp>git clone https://someuser@bitbucket.org/somcomp/someproj.git
Cloning into 'somproj'...
remote: Counting objects: 11762, done.
remote: Compressing objects: 100% (7028/7028), done.
Receiving objects: 100% (11762/11762), 48.36 MiB | 1.30 MiB/s, done.
remote: Total 11762 (delta 5706), reused 10135 (delta 4230)
Resolving deltas: 100% (5706/5706), done.
fatal: cannot create directory at 'someproj/node_modules/console-stamp/node_modules/dateformat/node_modules/meow/node_modules/normalize-package-data/node_modules/validate-npm-package-license/node_modules/spdx-correct/node_modules': No such file or directory
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry the checkout with 'git checkout -f HEAD'

C:\tmp>git status
fatal: Not a git repository (or any of the parent directories): .git
```

I don't see any spaces or unfriendly characters in my bitbucket repository. 
This is a zipped repo without some irrelevant folders: https://drive.google.com/file/d/0B28b26-yredPMU9FWUVwZjAwNGs/view?usp=sharing

**Comment by dscho** [comment]
It seems that this is a really long path. Have you tried to call `git config --global core.longpaths true` before cloning? This is one of the [known issues](https://github.com/git-for-windows/build-extra/blob/master/ReleaseNotes.md#known-issues) we describe in all of our release notes...

**Comment by stiv-yakovenko** [comment]
Yes, I tried git config --global core.longpaths true and git config --system core.longpaths true, both of them don't help.

**Comment by whoisj** [comment]
@stiv-yakovenko pls try v2.12.2(2), as a long path bug was fixed in 2.12.1.

**Comment by stiv-yakovenko** [comment]
@whoisj, tons of thanx!!
